plugins/modules/data_inputs_networks.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# https://github.com/ansible/ansible/issues/65816
+# https://github.com/PyCQA/pylint/issues/214
+
+# (c) 2022, Pranav Bhatt (adpranavb2000@gmail.com)
```

Comment:
This needs to be updated to RH like [here](https://github.com/ansible-collections/cisco.asa/blob/main/plugins/modules/asa_acls.py#L3), your name would come under author.